### DISCUSSION TITLE
Language-Fallback on custom translations

### DISCUSSION THREAD

**Discussion by Dominic-Marcelino** [directus/directus#24944] [Feature Requests]
### Describe the Improvement

When a custom translation is missing for the current language the key is displayed. This could be optimized in two ways:

1. Fallback to same base-language (e.g if we're on `en-US` but only have a translation vor `en-GB` this would be better then nothing)
2. Fallback to projects default language 
3. As a last alternative use the translation-key

**Comment by rijkvanzanten** [comment]
@Dominic-Marcelino Wanna open a PR for this? I believe this is something we can configure on `vue-i18n` globally rather than having to implement it separately for the custom translations https://vue-i18n.intlify.dev/guide/essentials/fallback.html#implicit-fallback-using-locales

**Comment by Dominic-Marcelino** [comment]
@rijkvanzanten `vue-i18n` implict fallback requires a base-locale version. For example if we register `en-US` as `en`, `en-GB` will automatically fallback to `en`.

This would require to rename a base-version for each lang and also a migration for custom translations. I assume this would also require some changes in Crowdin. 

Would you like to take this way, or should we define a [fallback-matrix](https://vue-i18n.intlify.dev/guide/essentials/fallback#explicit-fallback-with-decision-maps)?

Haven't fully checked the custom translation but I assume this would need some changes in the translation store to fetch the fallbacks.

**Comment by rijkvanzanten** [comment]
> This would require to rename a base-version for each lang and also a migration for custom translations. I assume this would also require some changes in Crowdin.

That feels risky..! We know the supported languages themselves don't really change all that often (if at all in recent history), so a static fallback language matrix should work for this use case!
